server/plugin.go

```diff
@@ -29,12 +30,19 @@ type externalConfig struct {
 	Secret string `json:"secret"`
 }
 
+const currentInstanceTTL = 1 * time.Second
```

Comment:
What does multiplying by one achieve?

Reply:
Just a no-op to make he code more readable.